index.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
   improve: 'apostrophe-pieces',
   collectionName: 'aposPiecesImportJobs',
 
-  afterConstruct: function(self, callback) {
+  afterConstruct: function (self, callback) {
```

Comment:
Please shut off whitespace transformations when submitting PRs.

Reply:
This one was on purpose in a separate commit, but I can refrain.

Reply:
That one isn't the convention in apostrophe itself, although it appears to be the new convention in-house.